app/boka-film/page.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import BookingForm from '@/app/components/BookingForm';
+import { useSearchParams } from 'next/navigation';
+import React from 'react';
+
+const BookMoviePage: React.FC = () => {
+  const searchParams = useSearchParams();
```

Comment:
I think there is a problem with this way of getting the information about a screening that you are doing. You can actually create fake screenings by altering the URL manually and then booking it to our database. The way to avoid this would be to only add the screenings _id as a search parameter and then fetch the relevant information from the database inside you boka-film/page.tsx. I also think it would make more sense to get the poster information in that same database fetch instead of fetching it already in the screenings component on the startpage like you have done now, since the poster is not used there. Feel like that makes the code clearer and maybe improves its separation of concerns.

![image](https://github.com/JrSchoolAccount/Kino-site-nextjs/assets/145323995/e01631bb-b141-463b-bc73-ff42af335331)